### PR TITLE
Specify app sharing dscp class for mac

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8887

confirmed with Teams team

### CODE REVIEW

File: Teams/QoS-in-Teams.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ Be aware of the following when you use these settings:
 
 - All clients, including mobile clients and Teams devices, will use these port ranges and will be affected by any DSCP policy you implement that uses these source port ranges. The only clients that will continue to use dynamic ports are the browser-based clients (clients that let participants join meetings by using their browsers).
 
-- Although the Mac client uses the same port ranges, it also uses hard-coded values for audio (EF) and video (AF41). These values aren't configurable.
+- Although the Mac client uses the same port ranges, it also uses hard-coded values for audio (EF), video and application/screen sharing (AF41). These values aren't configurable.
```
Comment (JohanFreelancer9) [inline Teams/QoS-in-Teams.md:149]:
```suggestion
- Although the Mac client uses the same port ranges, it also uses hard-coded values for audio (EF), video, and application/screen sharing (AF41). These values aren't configurable.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Teams/QoS-in-Teams.md (hunk 1 above)
Comment (get-itips) [reply Teams/QoS-in-Teams.md:149]: Is it clear that video AND app/screen sharing use AF41 if we add that comma? Thanks

Comment (JohanFreelancer9) [reply Teams/QoS-in-Teams.md:149]: Thanks, @get-itips, I was not sure and had to think about this one. Do you agree that we can replace the comma with "and"?

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@SerdarSoysal

**Pull request merged**